install_config/redeploying_certificates.adoc

```diff
@@ -329,8 +329,14 @@ file:
 openshift_master_ca_certificate={'certfile': '</path/to/ca.crt>', 'keyfile': '</path/to/ca.key>'}
 ----
 +
-If you do not set the above, then the current CA will be regenerated in the next
-step.
+If the CA certificate is issued by an intermediate CA, the bundled certificate must contain 
+the full chain (the intermediate and root certificates) for the CA in order to validate child certificates. 
++
+For example:
++
+# cat intermediate/certs/intermediate.cert.pem \
+      certs/ca.cert.pem > intermediate/certs/ca-chain.cert.pem
```

Comment:
you likely want to have this command read: 
```
# cat intermediate/certs/intermediate.cert.pem \
      certs/ca.cert.pem >> intermediate/certs/ca-chain.cert.pem
```

Using `>>` so that you append to the file, and not overwrite the file.